localstack/constants.py

```diff
@@ -158,21 +147,22 @@
 except Exception:
     MAX_POOL_CONNECTIONS = 150
 
-# credentials used in the test suite
-TEST_AWS_ACCESS_KEY_ID = "test"
-TEST_AWS_SECRET_ACCESS_KEY = (
-    "test"  # NOTE: In the near future, this will be set to a structured Access Key ID
-)
-
-# additional credentials used in the test suite (mainly for cross-account access)
-SECONDARY_TEST_AWS_ACCOUNT_ID = "000000000002"
-SECONDARY_TEST_AWS_ACCESS_KEY_ID = (
-    "000000000002"  # NOTE: In the near future, this will be set to a structured Access Key ID
-)
-SECONDARY_TEST_AWS_SECRET_ACCESS_KEY = "test2"
+# Fallback Account ID if not available in the client request
+DEFAULT_AWS_ACCOUNT_ID = "000000000000"
+
+# Credentials used in the test suite
+TEST_AWS_ACCOUNT_ID = os.getenv("TEST_AWS_ACCOUNT_ID") or DEFAULT_AWS_ACCOUNT_ID
+TEST_AWS_ACCESS_KEY_ID = os.getenv("TEST_AWS_ACCESS_KEY_ID") or "test"
+TEST_AWS_SECRET_ACCESS_KEY = os.getenv("TEST_AWS_SECRET_ACCESS_KEY") or "test"
+TEST_AWS_REGION_NAME = "us-east-1"
+
+# Additional credentials used in the test suite (when running cross-account tests)
+SECONDARY_TEST_AWS_ACCOUNT_ID = os.getenv("TEST_AWS_ACCOUNT_ID_2") or "000000000002"
+SECONDARY_TEST_AWS_ACCESS_KEY_ID = os.getenv("TEST_AWS_ACCESS_KEY_ID_2") or "000000000002"
```

Comment:
Notice the discrepancy between env var name and variable name.

I like the `_2` suffix for secondary creds more than the `SECONDARY_` prefix

I will refactor all the variable names in a follow up PR.